packages/beachball/src/git/index.ts

```diff
@@ -202,6 +198,22 @@ export function getCurrentHash(cwd: string) {
   return null;
 }
 
+/**
+ * Get the commit hash in which the file was first added.
+ */
+export function getFileAddedHash(filename: string, cwd: string) {
```

Comment:
we can call existing function `getCurrentHash` for getting the hash

Reply:
No, that's for getting the most recent commit hash, which might not be the one where the changefile was added.

Reply:
This is being run when the changelog is generated, not when the change file is generated